crates/typst/src/layout/grid/layout.rs

```diff
@@ -200,9 +210,9 @@ impl CellGrid {
         Self { cols, rows, cells, has_gutter, is_rtl }
     }
 
-    /// Resolves all cells in the grid before creating it.
-    /// Allows them to keep track of their final properties and adjust their
-    /// fields accordingly.
+    /// Resolves and positions all cells in the grid before creating it.
+    /// Allows them to keep track of their final properties and positions
+    /// adjust their fields accordingly.
```

Comment:
something is off in this sentence

Reply:
Git rebase moment :joy:
I'll fix, thanks